test/e2e/storage/persistent_volumes-local.go

```diff
@@ -132,8 +134,171 @@ var _ = framework.KubeDescribe("[Volume] PersistentVolumes-local [Feature:LocalP
+			pod2NodeName := podNodeName(pod2)
+			framework.Logf("Pod2 %q created on Node %q", pod2.Name, pod2NodeName)
+			Expect(pod2NodeName).To(Equal(node.Name))
+
+			_, writeErr := podExec(pod1, writeCmd)
+			Expect(writeErr).NotTo(HaveOccurred())
+			readOut, readErr := podExec(pod2, readCmd)
+			Expect(readErr).NotTo(HaveOccurred())
+
+			Expect(readOut).To(ContainSubstring(testVol.hostDir))
+
+			//delete pod2
+			framework.DeletePodOrFail(config.client, config.ns, pod2.Name)
+		})
+	})
+
+	Context("when two pods request one prebound PVC at the same time", func() {
+		var (
+			testVol *localTestVolume
+			node    *v1.Node
+		)
+
+		BeforeEach(func() {
+			// Choose the first node
+			node = &config.nodes[0]
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+			testVol = nil
+		})
+
+		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2", func() {
+			By("Initializing test volume")
+			testVol = setupLocalVolume(config, node)
+
+			By("Creating local PVC and PV")
+			createLocalPVCPV(config, testVol)
+			framework.Logf("PV created on node.Name %q", node.Name)
+
+			By("Creating a pod to write to the PV")
+			testFilePath := filepath.Join("/mnt/volume1", testFile)
+			writeCmd := fmt.Sprintf("echo %s > %s; cat %s", testVol.hostDir, testFilePath, testFilePath)
+			podSpec1 := createLocalPod(config, testVol, "")
+			pod1, pod1Err := config.client.CoreV1().Pods(config.ns).Create(podSpec1)
+			Expect(pod1Err).NotTo(HaveOccurred())
+
+			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, pod1))
+			pod1NodeName := podNodeName(pod1)
+			framework.Logf("Pod1 %q created on Node %q", pod1.Name, pod1NodeName)
+			Expect(pod1NodeName).To(Equal(node.Name))
+
+			By("Creating a pod to read from the PV")
+			readCmd := fmt.Sprintf("cat /mnt/volume1/%s", testFile)
+			podSpec2 := createLocalPod(config, testVol, "")
+			pod2, pod2Err := config.client.CoreV1().Pods(config.ns).Create(podSpec2)
+			Expect(pod2Err).NotTo(HaveOccurred())
+
+			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, pod2))
+			pod2NodeName := podNodeName(pod2)
+			framework.Logf("Pod2 %q created on Node %q", pod2.Name, pod2NodeName)
+			Expect(pod2NodeName).To(Equal(node.Name))
+
+			_, writeErr := podExec(pod1, writeCmd)
+			Expect(writeErr).NotTo(HaveOccurred())
+			readOut, readErr := podExec(pod2, readCmd)
+			Expect(readErr).NotTo(HaveOccurred())
+
+			Expect(readOut).To(ContainSubstring(testVol.hostDir))
+
+			framework.DeletePodOrFail(config.client, config.ns, pod1.Name)
+			framework.DeletePodOrFail(config.client, config.ns, pod2.Name)
+		})
+	})
+
 })
 
+// podNode wraps RunKubectl to get node where pod is running
+func podNodeName(pod *v1.Pod) string {
+	return strings.TrimSpace(framework.RunKubectlOrDie("get", "pod", pod.Name, fmt.Sprintf("--namespace=%s", pod.Namespace), "--no-headers", "-o=custom-columns=NODE:.spec.nodeName"))
```

Comment:
You can use the go client APIs to get the information instead of parsing text output.  `conflig.client.CoreV1().Pods(config.ns).Get(podName)`